main.go

```diff
@@ -36,7 +36,8 @@ import (
 
 // Constants and vars
 const version = "0.4.2"
-const workersCannelSize = 1024
+
+const workersCannelSize = 1
```

Comment:
if the workers dont pickup new work, we dont want to hide the fact that requests are piling up.